packages/health-check/src/notifyChanges.js

```diff
@@ -0,0 +1,47 @@
+const db = require("./db");
+const { sendMessageToHealthCheckChannel, getRoleByName } = require("./discord");
+
+const { PORTAL_NAME = "PORTAL_NAME not defined" } = process.env;
+
+const notificationTimestamps = { critical: 0, verbose: 0 };
+const notificationUnchangedInterval = 24 * 60 * 60 * 1000; // 24 hours
+
+function notifyChanges(type) {
+  const [currentEntry, previousEntry] = db.get(type).orderBy("date", "desc").take(2).value();
+
+  if (!currentEntry) return;
+
+  const currentPassing = currentEntry.checks.filter(({ up }) => up).length;
+  const currentTotal = currentEntry.checks.length;
+  const sendMessage = () => {
+    notificationTimestamps[type] = Date.now();
+
+    const message = `${PORTAL_NAME}: ${currentPassing} / ${currentTotal} ${type} checks passing`;
+
+    if (currentPassing === currentTotal) return sendMessageToHealthCheckChannel(message);
+
+    const role = getRoleByName("skynet-prod");
```

Comment:
I think we have to make all of the Skynet related things configurable. This script is not very useful to anyone besides us, unless they tweak it.